test-utils/fixtures/messages/ViewMessageMessages.scala

```diff
@@ -64,11 +64,18 @@ object ViewMessageMessages {
     val submitNewAlertRejectionLink = "submit a new alert or rejection"
   }
 
+  sealed trait IE813SubmissionFailureMessages {
+    _: i18n =>
```

Comment:
Internalisation isn't needed here, I see it's in other places, up to you if you want to remove it (and other references now)?